TypeCobol/Compiler/CupPreprocessor/CompilerDirectiveBuilder.cs

```diff
@@ -241,41 +241,76 @@ public void EnterSequenceNumberField(List<Token> seqNumField)
         {
             DeleteDirective deleteDirective = (DeleteDirective)CompilerDirective;
 
-            bool isFirst = true;
-            int previous = -42;
-            foreach(Token integerLiteralToken in seqNumField)
-            {
-                System.Diagnostics.Debug.Assert(integerLiteralToken.TokenType == TokenType.IntegerLiteral);
-                int current = (int)((IntegerLiteralTokenValue)integerLiteralToken.LiteralValue).Number;
 
-                if (isFirst)
+            foreach(Token token in seqNumField)
```

Comment:
missing space after 'foreach'

Reply:
Done in e705b29f87cea7244fae092d8bc7488e8de90876